EventFilter/DTRawToDigi/python/dturospacker_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+dturospacker = cms.EDProducer("DTuROSDigiToRaw",
+                                DTDigi_Source = cms.InputTag("simMuonDTDigis"),
+                                feds     = cms.untracked.vint32( 1369, 1370, 1371,),
```

Comment:
Those numbers are also hardcoded in ```DTuROSDigiToRaw::theCRT```: no need to make them configurable, as if you choose different ones you'll end up in some misbehaviour.
I think you should simply modify the producer removing the possibility to configure them